Appwrite/Appwrite_User_Authentication.ipynb

```diff
@@ -0,0 +1,390 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "6c4c63ae-5034-4150-8f7a-1541968c6913",
+   "metadata": {
+    "papermill": {},
+    "tags": []
+   },
+   "source": [
+    "<img width=\"10%\" alt=\"Naas\" src=\"https://landen.imgix.net/jtci2pxwjczr/assets/5ice39g4.png?w=160\"/>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8d64ced2-e7f3-442a-a3ec-8af66a99184e",
+   "metadata": {
+    "papermill": {},
+    "tags": []
+   },
+   "source": [
+    "# Appwrite - User Authentication"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "bf188fc8-9ea3-4853-ae06-06c3fe162226",
+   "metadata": {
+    "papermill": {},
+    "tags": []
+   },
+   "source": [
+    "**Tags:** #appwrite #python #sdk #authentication #user #create"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f122d69a-ae50-4d95-90df-fcb0d483fac7",
+   "metadata": {
+    "papermill": {},
+    "tags": []
+   },
+   "source": [
+    "**Author:** [SaiKiran M](www.linkedin.com/in/msaikiran9)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6f8a21c1-560a-405f-8599-fd906e95673d",
+   "metadata": {
+    "papermill": {},
+    "tags": []
+   },
+   "source": [
+    "**Last update:** 2023-10-08 (Created: 2023-10-08)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "96df94c1-842e-4588-a304-7b28a281138e",
+   "metadata": {
+    "papermill": {},
+    "tags": []
+   },
+   "source": [
+    "**Description:** This notebook will demonstrate how to use the Appwrite Python SDK to create a authentication and have options to `create_user`, `delete_user`, `list_all_users` as individual routines. It is useful for organizations that need to manage user authentication."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a14df074-778c-4f39-bc9e-c53d907dc830",
+   "metadata": {
+    "papermill": {},
+    "tags": []
+   },
+   "source": [
+    "**References:**\n",
+    "- [Appwrite Python SDK](https://github.com/appwrite/sdk-for-python)\n",
+    "- [Appwrite Documentation](https://appwrite.io/docs)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2bfe3fda-6f51-45b4-8dfa-06dda53622b2",
+   "metadata": {
+    "papermill": {},
+    "tags": []
+   },
+   "source": [
+    "## Input"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4f245a0a-f3a4-4750-be9c-752a07e7e4b5",
+   "metadata": {
+    "papermill": {},
+    "tags": []
+   },
+   "source": [
+    "### Import libraries"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "81730041-2523-462e-83c7-f3c08a3ca960",
+   "metadata": {
+    "papermill": {},
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "from appwrite.client import Client\n",
```

Comment:
Make sure to use a try-catch statement to overcome import error.

```python
try:
    from appwrite.client import Client
    from appwrite.services.users import Users
except ImportError:
    !pip install appwrite
    from appwrite.client import Client
    from appwrite.services.users import Users
```

Reply:
@srini047 Sure !